pkg/apis/pipeline/v1beta1/pipeline_validation_test.go

```diff
@@ -1288,6 +1306,45 @@ func TestFinallyTaskResultsToPipelineResults_Failure(t *testing.T) {
 			Message: `invalid value: referencing a nonexistent task`,
 			Paths:   []string{"spec.results[0].value"},
 		},
+	}, {
+		desc: "referencing nonexistent finally task result",
```

Comment:
is there a test case for a valid reference to a finally task result?

Reply:
added a test case